File: CPP/average.cpp
#include<iostream>
using namespace std;
int main() {
	int c,cpp,html,css,javascript;
	int total_Mark,average_Mark;
	cout<<"Enter your c mark:";
	cin>>c;
	cout<<"Enter your cpp mark:";
	cin>>cpp;
	cout<<"Enter your css mark:";
	cin>>css;
	cout<<"Enter your html mark:";
	cin>>html;
	cout<<"Enter your javascript mark:";
	cin>>javascript;

	total_Mark=javascript+c+cpp+css+html;
	average_Mark=total_Mark/5;
	cout<<"total mark:"<<total_Mark;
	cout<<"\naverage mark:"<<average_Mark;
	return 0;
}

File: JAVA/BASIC/average.java
package basic;
import java.util.Scanner;
public class average {
	public static void main(String[]agre)
	{
		int c,cpp,java,sql,java_Script;
		double total_Mark,total_Average;
		Scanner average=new Scanner(System.in);
		
		System.out.print("Enter your c mark:");
		c=average.nextInt();
		
		System.out.print("Enter your cpp mark:");
		cpp=average.nextInt();
		
		System.out.print("Enter your java mark:");
		java=average.nextInt();
		
		System.out.print("Enter your sql mark:");
		sql=average.nextInt();
		
		System.out.print("Enter your java script mark:");
		java_Script=average.nextInt();
		
		total_Mark=c+cpp+java+sql+java_Script;
		
		total_Average=total_Mark/5;
		
		System.out.println("total mark is:"+total_Mark);
		System.out.println("total average mark is:"+total_Average);
		
	}
	

}
